config/jobs/kubernetes/kops/kops-periodics-misc.yaml

```diff
@@ -114,3 +114,107 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-misc, kops-kubetest2
     testgrid-days-of-results: "30"
     testgrid-tab-name: kops-aws-kops-upgrade
+- interval: 1h # TODO(justinsb): reduce once working
+  name: e2e-kops-aws-upgrade-ko118-to-ko119
+  always_run: false
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-platform-aws: "true"
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  extra_refs:
+  - org: kubernetes
+    repo: kops
+    base_ref: master
+    workdir: true
+    path_alias: k8s.io/kops
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:latest-experimental
+      imagePullPolicy: Always
+      command:
+      - runner.sh
+      args:
+      - ./tests/e2e/scenarios/upgrade-ab/run-test.sh
+      env:
+      - name: CLOUD_PROVIDER
+        value: aws
+      - name: CLUSTER_NAME
+        value: e2e-f9b0e6b3f9-a6ae4.test-cncf-aws.k8s.io
+      - name: KOPS_STATE_STORE
+        value: s3://k8s-kops-prow
+      - name: KOPS_VERSION_A
+        value: 1.18.3
+      - name: KOPS_VERSION_B
+        value: 1.19.2
+      - name: K8S_VERSION_A
+        value: 1.18.18
+      - name: K8S_VERSION_B
+        value: 1.18.18
```

Comment:
I was wondering if we could somehow utilize channels so that we don't need to specify patch versions for all of these

Reply:
Using channels is a good idea.  I wasn't sure what we should be testing though e.g. 1.18.0 to 1.19.<latest> ?  So I punted on "how do we choose these values" problem ... for now!